### PR TITLE
move footer icons to CSS bg images

### LINKED ISSUE #2900: Move icons in footer to CSS background images
>     * The `ui/organisms/footer/index.tsx` is probably so big because it loads in all the SVGs. But why aren't

@schalkneethling 
Would it be feasible to move the SVG from the JS code into external images? I know it becomes hard to control styling then but it would be a win since, as images, they're technically lazy loaded and the footer is way below the fold.

_Originally posted by @peterbe in https://github.com/mdn/yari/issues/2877#issuecomment-777788131_

### PR DESCRIPTION
**Pull request by sohamsshah**
Fixes #2900 

Thank you MDN team for helping me out with this PR ❤ making OSS cakewalk for me haha ;)
Anyways, please review and let me know if any changes are needed, will be active with it.

### CODE REVIEW

File: client/src/ui/organisms/footer/index.tsx [hunk 1]
```diff
@@ -76,22 +71,22 @@ export function Footer() {
...
             <li>
               <a
+                className="social-icon twitter"
                 href="https://twitter.com/mozdevnet"
                 target="_blank"
                 rel="noopener noreferrer"
-                aria-label="Twitter"
               >
-                <TwitterIcon />
+                <span className="visuall-hidden">MDN on Twitter</span>
```
Comment (peterbe) [inline client/src/ui/organisms/footer/index.tsx:79]: Typo? `visually-hidden`

Comment (sohamsshah) [reply client/src/ui/organisms/footer/index.tsx:79]: Yeah, let me change it to Virtually. Btw let me know if other changes are needed as well :)

File: client/src/ui/organisms/footer/index.scss [hunk 2]
```diff
@@ -47,16 +47,35 @@
...
+  }
+
+  .instagram::before {
+    background: transparent url("~@mdn/dinocons/social/instagram.svg") 0 0
+      no-repeat;
+  }
+
+  .visuall-hidden {
+    display: none;
   }
```
Comment (schalkneethling) [inline client/src/ui/organisms/footer/index.scss:79]: I don't believe you will need to add this. The utility class should already be exposed by minimalist. On the relevant HTML element you should be able to add `className="visually-hidden"` and it should just work.

File: client/src/ui/organisms/footer/index.tsx (hunk 1 above)
Comment (schalkneethling) [reply client/src/ui/organisms/footer/index.tsx:79]: Yup, as Peter mentioned, you can use the existing `visually-hidden` class here and the other places later in the file.

**Comment by sohamsshah** [comment]
Done with the changes 👍

**Comment by peterbe** [comment]
@sohamsshah Can you merge in the latest `main` and push? (no rebase please)
Because, confusingly, the `client/build/static/js/main.[hash].chunk.js` file is currently larger as of this branch.

**Comment by sohamsshah** [comment]
Done with the changes, Please check :)

**Review by schalkneethling — APPROVED** [review]
🎉  ❤️  Approved! Thank you so much for your valued contribution @sohamsshah

**Comment by Ryuno-Ki** [comment]
>  The base branch requires all commits to be signed. [Learn more about signing commits.](https://docs.github.com/articles/about-gpg/)

Do you know how to handle this, @sohamsshah?

**Comment by sohamsshah** [comment]
> > The base branch requires all commits to be signed. [Learn more about signing commits.](https://docs.github.com/articles/about-gpg/)
> 
> Do you know how to handle this, @sohamsshah?

Interesting. Can you please brief me more about how to sign commits? Or send some resource to read about? 

Thanks.

**Comment by Ryuno-Ki** [comment]
I copied GitHub's link to https://docs.github.com/en/github/authenticating-to-github/about-commit-signature-verification#gpg-commit-signature-verification (without the anchor for the above comment).

Basically, it makes you create a GPG key (outlined in the above link) and upload it to GitHub.
This way, we can cryptographically verify, that you are the true author of the changes you made here.
(That's important since there were attacks in other projects like PHP. A commit was created seemingly by a maintainer, but contained malware. Since there was no verified commits process, it went unnoticed for a while).

If you make changes within GitHub's web interface, GitHub will sign your changes for you.
This will make a commit have a „Verified” badge in the UI here.

**Review by peterbe — APPROVED** [review]

**Comment by peterbe** [comment]
@sohamsshah Well done!! See if you can find the GitHub docs about how to set up git commit signing. It's a pain the back but you hopefully only have to do it once. 

Now, we need a more challenging bug for you to solve. :)
Got anything in mind?

**Comment by sohamsshah** [comment]
Thank you so much @peterbe and @schalkneethling ! 
Yes, I will setup the git commit signing for sure. Haha, yes sure; up for newer challenges to solve MDN ❤

I will have a look at the existing issues. Curious, if you can also suggest me some good issue, so that I can start working on it :)

**Comment by schalkneethling** [comment]
> Thank you so much @peterbe and @schalkneethling !
> Yes, I will setup the git commit signing for sure. Haha, yes sure; up for newer challenges to solve MDN ❤
> 
> I will have a look at the existing issues. Curious, if you can also suggest me some good issue, so that I can start working on it :)

What technologies are you most interested in or excited to work with?

**Comment by sohamsshah** [comment]
I am mostly into Frontend Development and love to explore component designs and architectures. Though, anything involving JS would excite me equally.

**Comment by peterbe** [comment]
> I will have a look at the existing issues. Curious, if you can also suggest me some good issue, so that I can start working on it :)

https://github.com/mdn/yari/issues?q=is%3Aopen+is%3Aissue+label%3Afrontend+no%3Aassignee is a list of front-end issues that are not already assigned.

**Pull request merged**